plugins/community/mypy-protobuf-grpc/v3.4.0/Dockerfile

```diff
@@ -0,0 +1,14 @@
+# syntax=docker/dockerfile:1.4
+FROM python:3.11.0-alpine3.17 AS build
+WORKDIR /app
+RUN python -mvenv /app
+ADD /requirements.txt requirements.txt
+RUN source ./bin/activate \
+ && pip install --no-cache-dir -r requirements.txt \
+ && pip uninstall --yes pip setuptools \
+ && rm -f requirements.txt bin/activate.fish bin/activate.csh bin/Activate.ps1
+
+FROM python:3.11.0-alpine3.17
```

Comment:
The distroless python package is still marked experimental so using alpine for now.